files/docker/grest/docker-entrypoint.sh

```diff
@@ -1,26 +1,25 @@
 #!/usr/bin/env bash
 
 ###################### Customisations - START ##################################
-apt-get update  > /dev/null 2>&1
-apt-get install -y socat curl gawk jq sudo  > /dev/null 2>&1
-apt-get install -y --no-install-recommends cron  > /dev/null 2>&1
+apt-get update >/dev/null 2>&1
+apt-get install -y socat curl gawk jq sudo >/dev/null 2>&1
+apt-get install -y --no-install-recommends cron >/dev/null 2>&1
 
-# To add a user without a password the command is the foloowing: 
+# To add a user without a password the command is the following:
 # adduser --disabled-password --gecos '' guild#
-sed -i 's/%sudo.*/%sudo   ALL=(ALL) NOPASSWD:ALL/g' /etc/sudoers > /dev/null 2>&1
-adduser postgres sudo  > /dev/null 2>&1
-
-
-# Sets the postgres user cronJobs
-chown postgres:root /etc/cron.d/crontab   > /dev/null 2>&1
-chmod 0660 /etc/cron.d/crontab   > /dev/null 2>&1
-touch /var/log/cron.log > /dev/null 2>&1
-chmod 0660 /var/log/cron.log > /dev/null 2>&1
-chown postgres:root /var/log/cron.log > /dev/null 2>&1
-crontab -u postgres /etc/cron.d/crontab  > /dev/null 2>&1
-
-# Listen for metrics via postgres user
-sudo -u postgres socat TCP-LISTEN:8059,reuseaddr,fork SYSTEM:"echo HTTP/1.1 200 OK;SERVED=true bash /getmetrics.sh " &
+sed -i 's/%sudo.*/%sudo   ALL=(ALL) NOPASSWD:ALL/g' /etc/sudoers >/dev/null 2>&1
+adduser "${POSTGRES_USER}" sudo >/dev/null 2>&1
+
+# Sets the ${POSTGRES_USER} user cronJobs
+chown "${POSTGRES_USER}":root /etc/cron.d/crontab >/dev/null 2>&1
+chmod 0660 /etc/cron.d/crontab >/dev/null 2>&1
+touch /var/log/cron.log >/dev/null 2>&1
+chmod 0660 /var/log/cron.log >/dev/null 2>&1
+chown "${POSTGRES_USER}":root /var/log/cron.log >/dev/null 2>&1
+crontab -u "${POSTGRES_USER}" /etc/cron.d/crontab >/dev/null 2>&1
+
+# Listen for metrics via ${POSTGRES_USER} user
+su "${POSTGRES_USER}" -c 'socat TCP-LISTEN:8059,reuseaddr,fork SYSTEM:"echo HTTP/1.1 200 OK;SERVED=true bash /getmetrics.sh "' &
```

Comment:
This is because the ENV variables should be available to the `getmetrics.sh` execution. It works and exports metrics but on entrypoint I get this message:
```
postgres_1         | PostgreSQL Database directory appears to contain a database; Skipping initialization
postgres_1         |
postgres_1         | Password: 2021-12-01 13:42:31.636 UTC [1] LOG:  starting PostgreSQL 13.5 (Debian 13.5-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
postgres_1         | 2021-12-01 13:42:31.638 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
postgres_1         | 2021-12-01 13:42:31.639 UTC [1] LOG:  listening on IPv6 address "::", port 5432
postgres_1         | 2021-12-01 13:42:31.672 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
postgres_1         | 2021-12-01 13:42:31.686 UTC [424] LOG:  database system was shut down at 2021-12-01 13:42:20 UTC
postgres_1         | 2021-12-01 13:42:31.698 UTC [1] LOG:  database system is ready to accept connections
postgres_1         | su: Authentication failure
```

Reply:
so we can remove sudo?

Reply:
I think we don't need sudo if we configure everything in this script. Because `root` runs this script?